### DISCUSSION TITLE
My column Time Stamp from an Enhanced Table doesn't reflect my timezone

### DISCUSSION THREAD

**Discussion by GloriaVazquezVidal** [fbaligand/kibana-enhanced-table#279] [Q&A]
Hi everybody and thank you a lot for answering me.

It's Gloria and I have this problem.

First of all, I have created an **enhanced table** in which one of my **columns in Data\Field is a timestamp** object.

![image](https://user-images.githubusercontent.com/40339528/185389112-92550d62-0d97-48e6-bde3-61fe9c96744c.png)

Secondly, my **current local timezone is GMT+2** and in Management/Advanced Settings is set "Timezone for date formatting" as "Browser", which means that my timestamps objects are displayed with 2 more hours (what is fine to me!). For example, as in this image from Discover:

![image](https://user-images.githubusercontent.com/40339528/185388119-0cd7bb28-276a-44eb-b781-c299af9c566c.png)

**Note that my primary key (PK) keeps my timestamp in UTC.**

So if you look at this timestamp, it reflects **"Aug 7, 2022 @ 01:46:55.000"** (perfect!).

Nevertheless, **in my enhanced table, my column TimeStamp doesn't reflect my local timezone (GMT+2)** as displays **"Aug 6, 2022 @ 23:46:55.000"** as all you can see in this image (following the same row of data):

![image](https://user-images.githubusercontent.com/40339528/185390003-48174561-ff13-4a14-ba4f-a48a8552bfc8.png)

**I reckon that it is supposed to be displayed in UTC timezone**, but I am not sure. **I would like to know if it is shown correctly as UTC timezone** (because you wanted to be like that) or if it is my fault and it should be shown in GMT+2 timezone.

Thank you in advance!
Gloria

**Comment by fbaligand** [comment]
Hi,

First, given your screenshot, it seems that you use "Document Table" visualization, and not "Enhanced Table".

Then I can confirm you that date should be rendered using browser timezone, as "Discover" app does.

I tried to reproduce your issue, but it works fine for me.
I mean: despite the date is stored in Elasticsearch with UTC timezone, it is rendered in "Document Table" viz with my browser timezone (that is exactly the same than you: UTC+2).

=> Could you tell what is your Kibana version and enhanced-table plugin version?
